src/Reports/Codeclimate.php

```diff
@@ -0,0 +1,340 @@
+        "PSR1.Classes.ClassDeclaration.MissingNamespace" => 600000,
+        "PSR1.Classes.ClassDeclaration.MultipleClasses" => 400000,
+        "PSR1.Files.SideEffects.FoundWithSymbols" => 500000,
+        "PSR2.ControlStructures.ControlStructureSpacing.SpacingAfterOpenBrace" => 50000,
+        "PSR2.ControlStructures.ElseIfDeclaration.NotAllowed" => 50000,
+        "PSR2.ControlStructures.SwitchDeclaration.TerminatingComment" => 50000,
+        "PSR2.Files.ClosingTag.NotAllowed" => 50000,
+        "PSR2.Files.EndFileNewline.NoneFound" => 50000,
+        "PSR2.Methods.FunctionCallSignature.CloseBracketLine" => 50000,
+        "PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket" => 50000,
+        "PSR2.Methods.FunctionCallSignature.Indent" => 50000,
+        "PSR2.Methods.FunctionCallSignature.MultipleArguments" => 70000,
+        "PSR2.Methods.FunctionCallSignature.SpaceAfterOpenBracket" => 50000,
+        "PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket" => 50000,
+        "PSR2.Methods.FunctionCallSignature.SpaceBeforeOpenBracket" => 50000,
+        "Squiz.Classes.ValidClassName.NotCamelCaps" => 50000,
+        "Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace" => 50000,
+        "Squiz.ControlStructures.ControlSignature.SpaceAfterCloseBrace" => 50000,
+        "Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis" => 50000,
+        "Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword" => 50000,
+        "Squiz.ControlStructures.ControlSignature.SpaceBeforeSemicolon" => 50000,
+        "Squiz.ControlStructures.ForLoopDeclaration.NoSpaceAfterFirst" => 50000,
+        "Squiz.ControlStructures.ForLoopDeclaration.NoSpaceAfterSecond" => 50000,
+        "Squiz.Functions.FunctionDeclarationArgumentSpacing.NoSpaceBeforeArg" => 50000,
+        "Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine" => 50000,
+        "Squiz.Functions.MultiLineFunctionDeclaration.ContentAfterBrace" => 50000,
+        "Squiz.WhiteSpace.ScopeClosingBrace.ContentBefore" => 50000,
+        "Squiz.WhiteSpace.ScopeClosingBrace.Indent" => 50000,
+        "Squiz.WhiteSpace.SuperfluousWhitespace.EndLine" => 50000,
+        "WordPress.Arrays.ArrayAssignmentRestrictions" => 50000,
+        "WordPress.Arrays.ArrayDeclaration" => 50000,
+        "WordPress.Arrays.ArrayKeySpacingRestrictions" => 50000,
+        "WordPress.CSRF.NonceVerification" => 50000,
+        "WordPress.Classes.ValidClassName" => 50000,
+        "WordPress.Files.FileName" => 50000,
+        "WordPress.Functions.FunctionRestrictions" => 50000,
+        "WordPress.NamingConventions.ValidFunctionName" => 50000,
+        "WordPress.NamingConventions.ValidVariableName" => 50000,
+        "WordPress.PHP.DiscouragedFunctions" => 50000,
+        "WordPress.PHP.StrictComparisons" => 50000,
+        "WordPress.PHP.StrictInArray" => 50000,
+        "WordPress.PHP.YodaConditions" => 50000,
+        "WordPress.VIP.AdminBarRemoval" => 50000,
```

Comment:
This looks like CodeClimate is using an old and outdated version of WPCS. Those `VIP` sniffs don't exist anymore.

Reply:
Actually, looks like all dependencies are way out of date. Last update of their `composer.json`/`composer.lock` is from sept 2018.